src/polkit/polkitunixprocess.c

```diff
@@ -739,7 +739,8 @@ polkit_unix_process_get_pid (PolkitUnixProcess *process)
       if (pid > 0)
         return pid;
 
-      g_error_free (error);
```

Comment:
This is correct, but we should apply the same logic to the function below: `polkit_unix_process_get_ppid` There is also an unguarded `g_error_free (error);` (line 772).